component-playground/config.js

```diff
@@ -11,7 +11,17 @@ var resolvePath = function(userPath) {
   return path.resolve(cwd, userPath);
 };
 
-module.exports = {
+var config;
+
+try {
+  config = require(
+    resolvePath(argv.configPath || 'component-playground.config')
```

Comment:
I think it is not necessary  to specify extension when requiring module in common js.